src/components/ViewSpecRegistrationModal.vue

```diff
@@ -238,6 +239,22 @@ export default defineComponent({
       }[currentState.value.matches('success_application_status_is_pending') ? 'success' : 'default']
     })
 
+    const authStrategyId = computed(() => {
+      const latestVersion = props.product.versions[props.product.version_count - 1]
```

Comment:
Played around testing this - noticed that this shouldn't be the latest version. It should be based off `route.params.product_version` - which is the product version id that you are currently viewing.